output/rust.geojson

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
This file should be ignored but doesn't seem to match the pattern in `.gitignore`

Reply:
I added an ignore rule for `${top_srcdir}/rust/case2geojson/output.geojson`.  This added file is `${top_srcdir}/output/rust.geojson`, and the CI does require these be tracked.  That's what I'm seeing with the `${top_srcdir}/.gitignore` currently in `main`.  Is there something I missed?